css/electrode.scss

```diff
@@ -1571,10 +1600,15 @@ img.footer-sub-logo {
 }
 
 @media (max-width: 767px) {
+    .row{
```

Comment:
@daversm I think we should not be overriding bootstrap styles. I think it's better to add your own classes to override the styles.